<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>courseChoose</class>
 <widget class="QWidget" name="courseChoose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>700</width>
     <height>250</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>250</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>741</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>华文中宋</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>可选课程预览</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="searchType">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>3</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentText">
       <string>按课程名查询</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>按课程名查询</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>按教师名查询</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="searchText">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12px</string>
      </property>
      <property name="placeholderText">
       <string>请输入</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="search">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="return_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>380</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认选课</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcom">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>5</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>475</y>
     <width>700</width>
     <height>250</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>250</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>已选课程预览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dropCourse">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>480</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退课</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#228974</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>24</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>鸭山大学</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>17</pointsize>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white
</string>
    </property>
    <property name="text">
     <string>教务系统</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>60</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white
</string>
    </property>
    <property name="text">
     <string>当前位置：首页 &gt; 选课</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Himalaya</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>Single-duck University</string>
    </property>
   </widget>
   <widget class="QToolButton" name="exit">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>60</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>返回首页</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
